### DISCUSSION TITLE
Fixit Theme Version

### DISCUSSION THREAD

**Discussion by shekhuverma** [hugo-fixit/FixIt#326] [Q&A] [answered]
How to check for which version of fixit theme i am using?

**Comment by Lruihao** [comment, marked as answer]
1. Check at theme footer
<img width="372" alt="image" src="https://user-images.githubusercontent.com/33419593/236399415-d8ba6c23-5407-44c0-a45c-eda2f7087682.png">

2. Check at html `head` tag of home page
<img width="567" alt="image" src="https://user-images.githubusercontent.com/33419593/236399706-0d43dc59-ed61-4cd1-8c85-7d461b4fbee8.png">

**Reply by shekhuverma** [reply]
Thankyou so much

**Comment by Lruihao** [comment]
`console.log(config.version)`
